src/main/java/Duke.java

```diff
@@ -1,10 +1,127 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    /**
+     * Process message and deem if user wants to quit.
+     * Print good bye message if user wants to quit.
```

Comment:
Use singular, present tense for verbs describing what the method does in Javadocs